### ISSUE TITLE
Repsonse Header Names in Mock Server are wrong

### ISSUE THREAD

**Issue by uweschaefer** [mulesoft/api-designer#472]
Please forgive me in case this is not the right place to open an issue for the mock service.

API Designer Version: 0.4.0
Browser: any
OS: any

Adding response headers to GET responses like

```
responses:
 200:
  headers:
   foo:
     default: 'bar'
     type: string
```

shows up in the AWS based mock service as 

`0: bar`

Looks like the name "foo" is replaced by some sort of array index? (next one will be named "1")

**Comment by nmarinel01** [comment]
Hi @uweschaefer ,
thanks for the report. Reproducible.

**Comment by carowright** [comment]
Hi @uweschaefer!
This is fixed and available from version 0.4.9.
